source/guides/maintenance/enable-systemd-boot-menu.rst

```diff
@@ -0,0 +1,22 @@
+.. _enable-systemd-boot-menu:
+
+Enable systemd-boot Menu
+########################
+
+The default installation of |CL| does not set a timeout value for the systemd-boot 
+bootloader. At boot time, you will not be presented with the systemd-boot menu. Without 
+a menu, you cannot interact with systemd-boot such as selecting a different kernel,
+editing kernel command line parameters, etc. 
+
+To set a timeout value, therefore, enabling the systemd-boot menu, follow these steps:
```

Comment:
Please simplify sentence: 
To set a timeout value to enable the systemd-boot menu...